minicriu-client.c

```diff
@@ -307,4 +336,72 @@ int minicriu_register_new_thread(void) {
 	return 0;
 }
 
+static int mc_getmap() {
+	char line[512];
+	mc_mapscnt = 0;
+	FILE *proc_maps;
+	proc_maps = fopen("/proc/self/maps", "r");
+
+	if (!proc_maps)
+		return 1;
```

Comment:
Here should be an error reporting to `stderr` (and in other places where we silently `return 1`).